tests/DevApps/WebAppCallsWebApiCallsGraph/TodoListService/appsettings.json

```diff
@@ -38,13 +45,13 @@
             }
         ]
     },
-    //"Kestrel": {
-    //    "Endpoints": {
-    //        "Http": {
-    //            "Url": "http://localhost:44351"
-    //        }
-    //    }
-    //},
+//    "Kestrel": {
```

Comment:
I could not find something in this file or in the README that mentions when to uncomment this. If there is such a time, could you point that out, or remove this if it is not needed?

Reply:
removed